docs/user-guide/reference/configuration/read-batch-size.md

```diff
@@ -0,0 +1,25 @@
+### Understanding `readBatchSize`
+
+At its core, `readBatchSize` is the **maximum number of messages a single vertex replica (pod) will try to read per cycle**. 
+
+
```

Comment:
Thinking about adding/linking some pointers regarding the parallelization with the new backend?
@vigith thoughts?